docs/general/builders-program.md

```diff
@@ -7,17 +7,26 @@ description:
 slug: ../builders-program
 ---
 
-The Substrate Builders Program is an initiative by Parity Technologies to take your Substrate
-project to the next level.
+Over 150 projects are using [Substrate](https://substrate.io) to create the next generation of
+blockchain infrastructure and application-specific chains.
+
+The [Substrate Builders Program](https://substrate.io/ecosystem/substrate-builders-program/) is an
+initiative by Parity Technologies to take your [Substrate](https://substrate.io) project to the next
+level.
 
 Projects building Substrate-based blockchains, applications, or ecosystem components can receive
 support for technical challenges, strategy, and ecosystem building from Parity’s extensive
 experience and resources.
 
-:::info
+:::info Square one initiative
```

Comment:
```suggestion
## Square One Initiative
```

I would suggest making this a header, that way we can also accomodate Nico's changes.